association_backend/association/domain/services/set_subscription_payed.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from datetime import datetime
+
+from association.domain.entities.subscription_entities import SubscriptionState
+from association.domain.exceptions import SubscriptionNotCreated
+from association.domain.repositories.association_repository import AssociationRepository
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class SubscriptionInputModel(BaseModel):
+    subscription_id: str
+
+
+async def set_subscription_payed(
```

Comment:
```suggestion
async def set_subscription_paid(
```

also would be great to try to have names more specific for our business logic 🤔

Reply:
I guess I have to go and stay in England for a month 🤦🏼 

By the way, do you prefer `associate_payment`, `register_payment` or `set_subscription_active`?

Reply:
For what do we use this service?

Reply:
this service is actually called by Webhhok when Stripe event 'customer.subscription.created' is fired, but I think it will be called when Stripe event`invoice.paid`, in the next commits